adhocracy4/comments_async/static/comments_async/comment.jsx

```diff
@@ -380,33 +380,34 @@ export default class Comment extends React.Component {
             />}
           <div className="a4-comments__action">
             <div className="row">
-              <div className="col-12 a4-comments__action-bar-container">
-                {this.renderRatingBox()}
-
-                <div className="a4-comments__action-bar">
-                  {((this.allowForm() && !this.props.is_deleted) || (this.props.child_comments && this.props.child_comments.length > 0)) &&
-                    <button className="btn btn--no-border a4-comments__action-bar__btn" type="button" onClick={this.toggleShowComments.bind(this)}>
-                      <a href="#child-comment-form">
-                        <i className={this.state.showChildComments ? 'fa fa-minus' : 'far fa-comment'} aria-hidden="true" /> {getAnswerForm(this.state.showChildComments, this.props.child_comments.length)}
-                      </a>
-                    </button>}
-
-                  {!this.props.is_deleted &&
-                    <a
-                      className="btn btn--no-border a4-comments__action-bar__btn" href={'?comment_' + this.props.id}
-                      data-bs-toggle="modal" data-bs-target={'#share_comment_' + this.props.id}
-                    ><i className="fas fa-share" aria-hidden="true" /> {translated.share}
-                    </a>}
-
-                  {!this.props.is_deleted && this.props.authenticated_user_pk && !this.props.is_users_own_comment &&
-                    <a
-                      className="btn btn--no-border a4-comments__action-bar__btn" href={'#report_comment_' + this.props.id}
-                      data-bs-toggle="modal"
-                    ><i className="fas fa-exclamation-triangle" aria-hidden="true" />{translated.report}
-                    </a>}
+              {!this.props.is_deleted &&
```

Comment:
putting this here means that if a comment is deleted the button to show the subcomments/replies is no longer visible, is that on purpose? as this would be a breaking change it would need to be coordinated with a+